configuration/packages.yaml

```diff
@@ -78,6 +78,6 @@ packages:
     buildable: False
     version: ['@0.10.0rc0']
     paths:
-      tensorflow+cuda@0.10.0rc0%gcc@5.3.0: /ssoft/spack/external/tensorflow/0.10.0rc0/tensorflow_venv_gpu 
+      tensorflow@0.10.0rc0+cuda@7.5.18+cudnn@4.0.cuda7%gcc@5.3.0: /ssoft/spack/external/tensorflow/0.10.0rc0/tensorflow_venv_gpu 
```

Comment:
Did you try the external package before committing? I think you want either to say:
```
tensorflow@0.10.0rc0 ^cuda@7.5.18 ^cudnn@4.0.cuda7%gcc@5.3.0: /ssoft/spack/external/tensorflow/0.10.0rc0/tensorflow_venv_gpu
```
or more likely:
```
tensorflow@0.10.0rc0+cuda%gcc@5.3.0: /ssoft/spack/external/tensorflow/0.10.0rc0/tensorflow_venv_gpu
```
When you ask **for variants** you don't need to have **versions** for them.